apps/test/unit/templates/rubrics/LearningGoalTest.jsx

```diff
@@ -146,17 +146,6 @@ describe('LearningGoal', () => {
     expect(wrapper.find('AiToken')).to.have.lengthOf(0);
   });
 
-  it('shows down arrow when closed and up arrow when open', () => {
```

Comment:
Since this was moved into the CSS, I didn't think this test was needed any more.